packages/content/src/Application/ContentNormalizer/Normalizer/SeoNormalizer.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Content\Application\ContentNormalizer\Normalizer;
+
+use Sulu\Content\Domain\Model\SeoInterface;
+
+class SeoNormalizer implements NormalizerInterface
+{
+    public function enhance(object $object, array $normalizedData): array
+    {
+        if (!$object instanceof SeoInterface) {
+            return $normalizedData;
+        }
+
+        $seoData = $object->getSeoData();
+        if (isset($seoData['seo'])) {
+            $normalizedData['seo'] = $seoData['seo'];
+        }
```

Comment:
Think this need to be:

```suggestion
        $normalizedData['seo'] = $seoData;
```

Reply:
changed in https://github.com/Prokyonn/sulu/pull/7